src/components/HistoryModal/Content.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import {
+  Loading,
+  Table,
+  Text,
+  Button,
+  Container,
+  Col,
+  Spacer,
+  Tooltip,
+} from '@geist-ui/react';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import { AmmDexOperation, AmmOrder, RefundOperation } from 'ergo-dex-sdk';
+import { toast } from 'react-toastify';
+import { exploreTx } from '../../utils/redirect';
+import { isRefundableOperation } from '../../utils/ammOperations';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faExternalLinkAlt, faUndo } from '@fortawesome/free-solid-svg-icons';
+import { useToggle } from '../../hooks/useToggle';
+import { ConfirmRefundModal } from '../ConfirmRefundModal/ConfirmRefundModal';
+import { truncate } from '../../utils/string';
+
+function renderOrder(
+  { status, txId, order }: AmmOrder,
+  open: boolean,
+  handleOpen: () => void,
+  handleClose: () => void,
+) {
+  return {
+    status,
+    txId: (
+      <CopyToClipboard text={txId} onCopy={() => toast.info('Copied')}>
```

Comment:
I think it's time to decorate all toasts with offset top to prevent their appearance over nav bar. top ~200px

Reply:
will do